src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/AdditionalFiles.BrowseObject.xaml

```diff
@@ -0,0 +1,61 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.-->
+<Rule
+  Name="AdditionalFiles"
+  DisplayName="Additional File"
+  PageTemplate="generic"
+  Description="File Properties"
```

Comment:
Please add PropertyPagesHidden="true" to all new "BrowseObjects" rules you added.